components/text_input/README.md

```diff
@@ -9,69 +9,65 @@ author: orinevares
 
 # Text Input
 
-Text inputs allow users to enter any combination of letters, numbers, or symbols of their choosing in a structured format.
+Text inputs allow users to enter a single line of any combination of letters, numbers, or symbols of their choosing in a structured format.
 
 ## Required
 
-## Last Updated: 2 days ago
+## Last Updated:
 
 ## Visual
-  This is where the visual goes
+  Text Input Visual:
+![Text Input](https://github.com/bcgov/design-system/blob/master/components/text_input/images/text_input.png?raw=true)
 
-## How to Use This:
-When users have to:
-* Input an answer that cannot be easily predicted
+## Requirements
+* This component must be used according the the standard outined in this document. Any feedback should be directed to the feedback forum.
+
+## Use This When
+* Users have to input an answer that cannot be easily predicted
 * Paste in a response
-*	Input text that is no longer than a single line
+* Input text that is no longer than a single line
 
+## Don't Use This when
+* A user needs to input multiple lines of text. Use a textarea component instead
+* Users can only choose from a selected list of answers
 
+## Design Protocol
+* The length of the text input should provide a hint to the user on how much text to write.
+* Text inputs must have visible labels aligned above the text input they refer to. They should be short, direct, and written in sentence case.
+* Allow copy and paste functionality
```

Comment:
"Allow copy and paste functionality."  - add a period at the end so that it is consistent with punctuation on the other lists